BP-TPSE-Projektgruppe-80/NaPyTau
# Added menubar and buttons to the GUI

### Summary

This adds a menu bar to the GUI, as well as checkboxes for selecting which datapoints to use for the fitting of the graph and the calculation of tau.
It also replaces the previous UI mockup.

---

### PR checklist

- [X] I have written tests that fail without my changes. _Or I did not add them on purpose._
- [X] I have included instructions on how to test this, assuming they are not already apparent from the linked issue.

---

## How to test

There are no end-to-end tests for the UI yet, therefore you can only check the GUI manually.
You can check if it is working properly by making sure it starts without errors.
All buttons should be able to be pressed without errors.
Buttons where the backend functionality is not yet implemented will at least give an output to the developer terminal.

### Closes/Fixes/Related to


* Closes #6
